script.service.kodi.callbacks/resources/language/English/strings.po

```diff
@@ -123,11 +126,14 @@ msgid "Command for Task %s, Event %s completed succesfully!"
 msgstr ""
 
 msgctxt "#32025"
-msgid "\nThe following message was returned: %s"
+msgid "\n"
+"The following message was returned: %s"
```

Comment:
See: [https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html](url) at end of page.
My understanding is that it is easier for translators.